kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/internal/ClusterOperationsImpl.java

```diff
@@ -17,29 +17,24 @@
 package io.fabric8.kubernetes.client.dsl.internal;
 
 import com.fasterxml.jackson.databind.ObjectMapper;
-import io.fabric8.kubernetes.client.Version;
 import io.fabric8.kubernetes.client.VersionInfo;
 import io.fabric8.kubernetes.client.Config;
 import io.fabric8.kubernetes.client.KubernetesClientException;
 import io.fabric8.kubernetes.client.dsl.base.OperationContext;
 import io.fabric8.kubernetes.client.dsl.base.OperationSupport;
 import io.fabric8.kubernetes.client.utils.URLUtils;
-import io.fabric8.openshift.api.model.ClusterVersionList;
 import okhttp3.OkHttpClient;
 import okhttp3.Request;
 import okhttp3.Response;
 
 import java.io.IOException;
-import java.net.HttpURLConnection;
 import java.text.ParseException;
 import java.util.HashMap;
 import java.util.Map;
 
 public class ClusterOperationsImpl extends OperationSupport {
-  private String versionEndpoint;
+  protected final String versionEndpoint;
   public static final String KUBERNETES_VERSION_ENDPOINT = "version";
-  public static final String OPENSHIFT_VERSION_ENDPOINT = "version/openshift";
-  public static final String OPENSHIFT4_VERSION_ENDPOINT = "apis/config.openshift.io/v1/clusterversions";
   public static final ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
We should remove this mapper and either user `Serialization.jsonMapper` or any of the appropriate helper methods in that class.